radioapp/zeroconf.py

```diff
@@ -0,0 +1,20 @@
+from zeroconf import ServiceBrowser, Zeroconf
+import threading
+
+
+class LmrListener(object): #declare this before calling PairingScan
+
+    def __init__(self):
+        self.stop_searching = threading.Event()
+
+    def add_service(self, zeroconf, type, name):
+        info = zeroconf.get_service_info(type, name)
+        print("Service %s added, service info: %s" % (name, info))
```

Comment:
There's no way to tell when a service goes away.

Reply:
Didn't think we needed to know, I can re-add that code for a closing since it was just a print statement.